cdk/lib/amiable/amiable.test.ts

```diff
@@ -1,18 +1,16 @@
 import { App } from "aws-cdk-lib";
 import { Template } from "aws-cdk-lib/assertions";
+import { codeProps, prodProps } from "../../bin/amiable";
 import { Amiable } from "./amiable";
 
 describe("The Amiable stack", () => {
   it("matches the snapshot", () => {
     const app = new App();
-    const stack = new Amiable(app, "amiable", {
-      migratedFromCloudFormation: true,
-      stack: "deploy",
-      stage: "CODE",
-      env: { region: "eu-west-1" },
-      domainName: "amiable.code.dev-gutools.co.uk",
-    });
 
-    expect(Template.fromStack(stack).toJSON()).toMatchSnapshot();
+    const codeStack = new Amiable(app, "amiable", codeProps);
```

Comment:
Updating `"amiable"` id here results in changes to the snapshot, so I have decided it's better to tolerate this inconsistency 😁